R/enhanceFeatures.R

```diff
@@ -50,7 +52,7 @@ NULL
 
 #' @importFrom assertthat assert_that
 .enhance_features <- function(X.enhanced, X.ref, Y.ref, 
-    feature_names = rownames(Y.ref), model = c("xgboost", "dirichlet", "lm")) {
+    feature_names = rownames(Y.ref), model = c("xgboost", "dirichlet", "lm"), tune.nrounds, train.n) {
```

Comment:
general note - I try to keep lines to 80 characters (also recommended in Bioconductor code guidelines) and so I typically wrap long function signatures like this